diff --git a/cmake/HPX_SetupApex.cmake b/cmake/HPX_SetupApex.cmake
index 2b3d071e50..f4719712ba 100644
--- a/cmake/HPX_SetupApex.cmake
+++ b/cmake/HPX_SetupApex.cmake
@@ -51,7 +51,7 @@ if(HPX_WITH_APEX AND NOT TARGET APEX::apex)
       include(FetchContent)
       fetchcontent_declare(
         apex
-        GIT_REPOSITORY https://github.com/UO-OACISS/apex.git
+        GIT_REPOSITORY https://github.com/timniederhausen/apex.git
         GIT_TAG ${HPX_WITH_APEX_TAG}
       )
 
